extensions/objects/protractor.js

```diff
@@ -30,11 +30,24 @@ var AlgebraicExpressionEditor = function(elem) {
       await waitFor.elementToBeClickable(
         elem, '"Algebraic Input" editor takes too long to be clickable');
       await elem.click();
-      var algebraicInputElem = element(by.css('.guppy_active'));
+      var algebraicInputElem = element.all(by.css(
+        '.protractor-test-guppy-div')).last();
       var present = await algebraicInputElem.isPresent();
       if (present) {
         await algebraicInputElem.sendKeys(text);
       }
+    },
+    getValue: async function() {
+      var algebraicInputElem = element.all(by.css(
+        '.protractor-test-guppy-div')).last();
+      var present = await algebraicInputElem.isPresent();
```

Comment:
Oops, sorry I missed this one earlier. The element might just have not loaded yet, so we need a waitFor just like the earlier comment you addressed

Reply:
Done.